packages/plugin-electron-process-info/procinfo.js

```diff
@@ -1,3 +1,6 @@
+// eslint-disable-next-line no-eval
+const isPreload = !!eval('typeof global !== "undefined"') && !!eval('typeof window !== "undefined"')
```

Comment:
This works for preload scripts because `global` is not defined in the browser but it is in a Node-like environment. However, when you bundle with webpack, it defines `global` for you, so it also returns `true` in a renderer bundle. The `eval()` solution is a workaround for this, but then it's not compatible with sensible CSP rules.